src/assets/css/decred-v5.css

```diff
@@ -1,29 +1,33 @@
 /* inconsolata-regular - latin */
+
 @font-face {
   font-family: 'Inconsolata';
   font-style: normal;
   font-weight: 400;
-  src: url('../fonts/inconsolata-v15-latin-regular.eot'); /* IE9 Compat Modes */
```

Comment:
The change set is huge because of a code reformat.  In the commit, please only include code that's relevant to the change, but no lines that haven't been modified (apart from the formatting).

Reply:
Thank for @peterzen, the change in question is as a result of my IDE code formatting settings. Nothing was changed apart from the reformatting.

Thank you.

Reply:
Can you commit this change without reformatting?

Reply:
Yes